src/components/RowOfCards.tsx

```diff
@@ -74,6 +74,12 @@ const RowOfCollectionCards: React.FunctionComponent<{
     cardSpecs["by-topic"] = useCollectionCardSpec(
         CollectionCardLayout.iconAndBookCount
     );
+    // Unexpected. But prevents a crash if it happens.
```

Comment:
Not as sure about this one, and it's a totally different topic... can you split it into its own commit?